rocAL/rocAL/CMakeLists.txt

```diff
@@ -118,13 +123,15 @@ if(NOT Threads_FOUND)
     set(BUILD_ROCAL false)
     message("-- ${Yellow}NOTE: rocAL library requires Threads, Not Found ${ColourReset}")
 endif()
+if(${libsnd_LIBS} STREQUAL libsnd_LIBS-NOTFOUND)
+    message("-- ${Yellow}NOTE: rocAL library requires libsnd(sndfile), Not found. Try to specify it's location with `-DLIBSND_ROOT_DIR`.${ColourReset}")
+endif()
+message(STATUS "Found libsnd: ${libsnd_LIBS}")
 
 if(${BUILD_ROCAL})
-    # AMD OpenVX & VX_RPP
-    set(LINK_LIBRARY_LIST ${LINK_LIBRARY_LIST} openvx vx_rpp)
-    # AMD RPP
+    # AMD OpenVX & RPP
     include_directories(${AMDRPP_INCLUDE_DIRS})
-    set(LINK_LIBRARY_LIST ${LINK_LIBRARY_LIST} ${AMDRPP_LIBRARIES})
+    set(LINK_LIBRARY_LIST ${LINK_LIBRARY_LIST} openvx vx_rpp)
```

Comment:
Is this change needed in this PR?

Reply:
This change was not reflected in the working branch, changed it according to Lakshmis base branch